fetchers/ecb/index.ts

```diff
@@ -0,0 +1,39 @@
+import * as exchangeRates from "ecb-euro-exchange-rates";
+import { Consola } from "consola";
+import { PriceDataFetched, Fetcher } from "../../types";
+
+const logger =
+  require("../../utils/logger")("fetchers/ecb") as Consola;
+
+const ecbFetcher: Fetcher = {
+  async fetchAll(symbols: string[]): Promise<PriceDataFetched[]> {
+    // Fetching prices
+    const { rates }: any = await exchangeRates.fetch();
+    const usdRate = rates.USD;
+
+    // Building prices array
+    const prices = [];
+    for (const symbol of symbols) {
+      if (rates[symbol] !== undefined) {
```

Comment:
How about: 
if (rates[symbol] !== undefined) {
  if (symbol === "EUR") {
          prices.push({
            symbol,
            value: usdRate,
          });
  } else {
        
    prices.push({
      symbol,
      value: (1 / rates[symbol]) * usdRate as number,
    });
  }

I'm afraid of doubling the EUR enty